### PR TITLE
populate subscription data for initial setup

### PR DESCRIPTION
**Pull request by kentwills**
Helps initial state for people creating the app for the first time #23

### CODE REVIEW

File: yelp_beans/routes/tasks.py [hunk 1]
```diff
@@ -92,3 +96,31 @@ def send_match_emails():
         logging.info(matches)
         send_batch_meeting_confirmation_email(matches, spec)
     return "OK"
+
+
+@tasks.route('/init', methods=['GET'])
+def init():
```
Comment (ajm188) [inline yelp_beans/routes/tasks.py]: Are we sure this should be its own route? I feel like this is better-suited to a one-off executable that we provide.

Similar to `rake db:seed` that rails does

File: yelp_beans/routes/tasks.py [hunk 2]
```diff
@@ -92,3 +96,31 @@ def send_match_emails():
         logging.info(matches)
         send_batch_meeting_confirmation_email(matches, spec)
     return "OK"
+
+
+@tasks.route('/init', methods=['GET'])
```
Comment (ajm188) [inline yelp_beans/routes/tasks.py]: And this definitely shouldn't be a GET

File: yelp_beans/routes/tasks.py (hunk 1 above)
Comment (kentwills) [inline yelp_beans/routes/tasks.py]: Explored the option of using a script with the [remote api](https://cloud.google.com/appengine/docs/standard/python/tools/remoteapi).  I was attempting to cut down on the user complexity of having to add `remote_api: on` to the app.yaml, deploying, running a script, removing the line, and deploying again.

The route is only accessible by admins and is a noop if there is data, while not great from an engineering perspective, it seems to be the quickest upstart from a user perspective.

File: yelp_beans/routes/tasks.py (hunk 2 above)
Comment (kentwills) [inline yelp_beans/routes/tasks.py]: agreed

**Comment by kentwills** [comment]
Looking for a better route to populate starter data.

**Pull request closed without merging**